### PR TITLE
[examples] Update README Files

### PR DESCRIPTION
**Pull request by mcsdevv**
This PR updates the `README.md` files with the following changes:

- Removal of CLI deployment.
- Amends Blitz.js to be in line with other `README.md` formats.

### CODE REVIEW

File: examples/amp/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Deploy your own AMP project with Vercel.
 
 [![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/import/project?template=https://github.com/vercel/vercel/tree/master/examples/amp)
 
-_Live Example: https://amp.now-examples.now.sh_
+_Live Example: https://amp-example.vercel.app_
```
Comment (styfle) [inline examples/amp/README.md]: This is a 404

Comment (mcsdevv) [reply examples/amp/README.md]: Updated!

**Review by styfle — COMMENTED** [review]
You should also update the `demo` property in `frameworks.json` for each framework to match the readme https://github.com/vercel/vercel/blob/b85a0a349af988f32f1e6d054c26f648f4e2808e/packages/frameworks/frameworks.json#L5

**Review by styfle — APPROVED** [review]
Great, thanks!

**Review by Timer — APPROVED** [review]

**Pull request merged**